### PR TITLE
Administrator - login page - improove accessiblity

### PR DESCRIPTION
**Pull request by ylahav**
Pull Request for Issue # .

### Documentation Changes Required

The login form was change to be accessible
- screen-reader only labels were added
- colours were changed to get more contrast

### CODE REVIEW

File: administrator/modules/mod_login/tmpl/default.php [hunk 1]
```diff
@@ -82,7 +83,7 @@ class="form-control input-full"
 
 		<div class="form-group">
 			<button class="btn btn-success btn-block btn-lg">
-				<span class="icon-lock icon-white"></span> <?php echo JText::_('MOD_LOGIN_LOGIN'); ?>
+				<span class="icon-lock"></span> <?php echo JText::_('MOD_LOGIN_LOGIN'); ?>
```
Comment (brianteeman) [inline administrator/modules/mod_login/tmpl/default.php]: Needs aria-hidden=true or a screen reader will read the unicode value of the icon

**Comment by C-Lodder** [comment]
@ylahav - you're also missing the label for the 2FA secret key input. Needs to go above this line: https://github.com/ylahav/joomla-cms/blob/1fea828baaf7468c8dfba2d6f0a5ff741f4b7f9c/administrator/modules/mod_login/tmpl/default.php#L66

**Comment by wilsonge** [comment]
We need to add the language string for `TPL_ATUM_GOTO_JOOMLA_HOME_PAGE` :)

**Comment by C-Lodder** [comment]
I also think any a11y CSS should go into a separate file (a11y.css). I'd then like to add a fixed button in the corner somewhere which when clicked on, adds a class to the body (`<body class="a11y">`) and all a11y CSS can be done like so:

```
.a11y .btn-success {
    color:  #000;
 }
```

File: administrator/templates/atum/login.php [hunk 2]
```diff
@@ -102,9 +102,12 @@
...
 			</li>
 			<li class="nav-item">
-				<a class="login-joomla hasTooltip" href="https://www.joomla.org" target="_blank" title="<?php echo JHtml::tooltipText('TPL_ATUM_ISFREESOFTWARE'); ?>"><span class="fa fa-joomla"></span></a>
+				<a class="login-joomla hasTooltip" href="https://www.joomla.org" target="_blank" title="<?php echo JHtml::tooltipText('TPL_ATUM_ISFREESOFTWARE'); ?>">
+					<span class="fa fa-joomla"></span>
+					<span class="sr-only"><?php echo JText::_('TPL_ATUM_GOTO_JOOMLA_HOME_PAGE'); ?></span>
+				</a>
 			</li>
-			<li class="nav-item">	
+			<li class="nav-item">	<span class="sr-only"><?php echo JText::_('TPL_ATUM_GOTO_JOOMLA_HOME_PAGE'); ?></span>
```
Comment (wilsonge) [inline administrator/templates/atum/login.php]: Do we need this twice on the page?

File: administrator/modules/mod_login/tmpl/default.php (hunk 1 above)
Comment (ylahav) [inline administrator/modules/mod_login/tmpl/default.php]: fine... will be done

File: administrator/templates/atum/login.php (hunk 2 above)
Comment (ylahav) [inline administrator/templates/atum/login.php]: no - it is a mistake - will be fixed

**Comment by brianteeman** [comment]
i am not in favour of having a separate a11y css OR having a setting to improve a11y. We should be creating an accessible template by default not having an option to be accessible or not. (If nothing else that will lead to the bad old hathor days)

**Comment by mbabker** [comment]
The only accessibility options should be related to how Hathor offered different color styles for different cases (like the high contrast option, also not saying core needs to offer that but just using that example).  Otherwise, accessibility is something you bake natively into the UI and markup, it's not an afterthought or an opt-in feature.

**Comment by brianteeman** [comment]
+100 lots not repeat the mistakes of the past!!!

**Comment by C-Lodder** [comment]
@brianteeman - Yup don't worry. We had a chat about it and won't be going down that route

File: administrator/templates/atum/language/en-GB/en-GB.tpl_atum.ini [hunk 3]
```diff
@@ -16,3 +16,4 @@ TPL_ATUM_XML_DESCRIPTION="Continuing the Egyptian god/goddess theme (Khepri from
 TPL_BACK_TO_CONTROL_PANEL="Back to Control Panel"
 TPL_ATUM_POST_INSTALLATION_MESSAGES="Post-installation messages"
 TPL_ATUM_PRIVATE_MESSAGES="Private Messages"
+TPL_ATUM_GOTO_JOOMLA_HOME_PAGE="Go to Joomla organization home page"
```
Comment (C-Lodder) [inline administrator/templates/atum/language/en-GB/en-GB.tpl_atum.ini]: organi**z**ation >>> organi**s**ation

**Comment by C-Lodder** [comment]
@ylahav - please compile the SCSS so others can test this PR

**Comment by wilsonge** [comment]
Merged with https://github.com/joomla/joomla-cms/commit/1aae03dd4586c1ee8c1f3f83deac0e47d0fd04c2

**Pull request closed without merging**